modules/imaging_browser/test/TestPlan.md

```diff
@@ -22,7 +22,8 @@
 
 ### C. Main panel:  per acquisition:
 
-16. Files can be downloaded (links clickable) only IFF has permission (future feature not implemented yet)
+16. Files can be downloaded (links clickable). Ensure that DICOM downloads are
+prepended with the Patient Name.
```

Comment:
Why did the test plan lost the permission check here?

Reply:
I meant to reword but must've forgotten.

Reply:
Oh I think I deleted this line because the permission checking exists now, from what I know. I think this should be a subset of the module's permission check and doesn't need to be explicitly mentioned here.

Reply:
I'm not sure where you're referring to.. I don't see anywhere else in the test plan that covers this and all the other sections in this test plan has explicit permission testing.

Reply:
I added it back